### ISSUE TITLE
Anbox support

### ISSUE THREAD

**Issue by Zizico2** [gamer-os/gamer-os#89]
I know this is a bit out there so don't hesitate to close this issue. 
I would really like to see anbox support on GamerOS. I have no idea how hard or easy it would be. It is currently distributed as a snap. 
My ISP has been serving TV over IP for a long time. But until a while ago we would need their receiver to decode it. 
Now they provide apps for Android, iOS, MacOS and Windows as a "no cords" solution. It is, honestly, the best TV service around (at least where I live). There is a huge thread on their forums asking or a Linux app or at least a Kodi add-on. 
For now I have been using Anbox on my media centre pc. This is the only feature preventing me from moving over to GamerOS :/

TLDR: I need anbox to watch TV on GamerOS. It's support is a deal breaker for me.

**Comment by moarblur** [comment]
It may be possible to run some Android games through anbox as well.

**Comment by alkazar** [comment]
Even ignoring the technical hurdles, this doesn't seem like something that would be usable with the SteamOS compositor.

If you really want to, you can edit `profiles/default` by adding `anbox-git` to the list of AUR packages and build your own custom GamerOS image.

Another alternative is to install GamerOS, run `sudo frzr-unlock`, disable the `frzr-autoupdate` service. Now you will be able to use your system as a regular Arch Linux system and install additional software with pacman at the expense of losing automatic updates.
